src/main/java/iudx/file/server/common/Constants.java

```diff
@@ -39,4 +42,15 @@ public class Constants {
 
   // RMQ queues and exchanges.
   public static String INVALID_SUB_Q = "fs-invalid-sub";
+  public static final String ITEM_TYPE_RESOURCE = "iudx:Resource";
+  public static final String ITEM_TYPE_RESOURCE_GROUP = "iudx:ResourceGroup";
```

Comment:
```suggestion
  public static final String ITEM_TYPE_RESOURCE_GROUP = "ResourceGroup";
```

It can be different for different deployment

Reply:
done